packages/classic-test-app/package.json

```diff
@@ -37,7 +37,7 @@
     "pretender": "3.4.7",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "4.0.2",
-    "ember-data": "~3.17.0",
+    "ember-data": "~4.4.0",
```

Comment:
✂️ 
Let's keep torii upgrade as the only change if possible

Reply:
The "default" test fails without this.